src/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# add MQT::qfr library
+#add_subdirectory("${PROJECT_SOURCE_DIR}/extern/qfr" "extern/qfr"
+#                 EXCLUDE_FROM_ALL)
+# add MQT::ddsim library
+add_subdirectory("${PROJECT_SOURCE_DIR}/extern/ddsim" "extern/ddsim"
+                 EXCLUDE_FROM_ALL)
+
+# add MQT::qcec library
+add_subdirectory("${PROJECT_SOURCE_DIR}/extern/qcec" "extern/qcec"
+                 EXCLUDE_FROM_ALL)
+
+#add_library(
+#  ${PROJECT_NAME}
+#  ${PROJECT_SOURCE_DIR}/include/checker
+#  ${PROJECT_SOURCE_DIR}/include/Configuration.hpp
+#  ${PROJECT_SOURCE_DIR}/include/EquivalenceCriterion.hpp
+#  ${PROJECT_SOURCE_DIR}/include/EquivalenceCheckingManager.hpp
+#  ${PROJECT_SOURCE_DIR}/include/ThreadSafeQueue.hpp
+#  EquivalenceCheckingManager.cpp
+#  checker/EquivalenceChecker.cpp
+#  checker/dd/DDEquivalenceChecker.cpp
+#  checker/dd/DDConstructionChecker.cpp
+#  checker/dd/DDSimulationChecker.cpp
+#  checker/dd/DDAlternatingChecker.cpp
+#  checker/dd/applicationscheme/GateCostApplicationScheme.cpp
+#  checker/dd/simulation/StateGenerator.cpp
+#  checker/zx/ZXChecker.cpp)
+
+add_library(
+    ${PROJECT_NAME})
```

Comment:
Does that even work when configuring CMake?

I would have guessed this needs to be something like
```cmake
add_library(${PROJECT_NAME} INTERFACE)
```

Reply:
Done in https://github.com/tyi1025/dd_eval/pull/1/commits/143bb197854bfda8eb4056f9b0b3c0e4c07b7998